adapters/tasmania.js

```diff
@@ -0,0 +1,143 @@
+'use strict';
+
+import { REQUEST_TIMEOUT } from '../lib/constants';
+import { default as baseRequest } from 'request';
+const request = baseRequest.defaults({timeout: REQUEST_TIMEOUT});
+import _ from 'lodash';
```

Comment:
Using the new cloning method referenced below, _.cloneDeep gets used, so keeping this in for now, that ok?